test/src/edu/stanford/nlp/io/IOUtilsFuzzTest.java

```diff
@@ -0,0 +1,108 @@
+package edu.stanford.nlp.io;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import java.util.Random;
+
+import static edu.stanford.nlp.io.IOUtils.*;
+
+/**
+ * Generate randomized input data for testing I/O operations.
+ *
+ * @author Julius Willems
+ */
+public class IOUtilsFuzzTest {
```

Comment:
There are two dependencies they have junit-quickcheck-core and junit-quickcheck-generators but they are using 0.5 version and current is 0.9. Maybe you would like to use those features provided by the dependencies

Reply:
Thanks @RinorSefa for this hint. From a first glance at these libraries I figured that they are used to test properties of Objects by the use of annotations. https://pholser.github.io/junit-quickcheck/site/0.7/index.html. So I got the impression, they aim at another angle.